bin/env.bash

```diff
@@ -21,4 +21,6 @@ export MODULE_ORIGINAL=${MODULE_ORIGINAL:-libtree_sitter_dyn.$ext}
 export MODULE_NAME=${MODULE_NAME:-tree-sitter-dyn}
 export MODULE_RENAMED=${MODULE_NAME}.so
 export MODULE_FULL="$MODULE_DIR/$MODULE_RENAMED"
-export EMACS=${EMACS:-emacs}
+if [ -z ${EMACS+x} ]; then ## $EMACS is not set
```

Comment:
This change is unnecessary. The current code already uses the env var, if set, defaulting to `emacs` otherwise.

Reply:
Oh, I misunderstood bash then 🤔